tooling/cli/src/mobile/init.rs

```diff
@@ -57,20 +56,7 @@ pub fn configure_cargo(
     }
   }
 
-  let mut dot_cargo = dot_cargo::DotCargo::load(app)?;
-  // Mysteriously, builds that don't specify `--target` seem to fight over
```

Comment:
Yeah but users would need to ignore `.cargo/config.toml` otherwise, it will be modified with different OSes and will make it hard for users with git. That's why I want to remove it from the CLI and move this piece to the the FAQ on the website.

Reply:
What is the change across different OSes? we should try to avoid overwriting it if it already has what we need.

Reply:
what this workaround does, is that is specifies a default build target, for example:
```toml
[build]
target = 'x86_64-pc-windows-msvc'
```
so if someone else working on the project, uses linux it will change to:
```toml
[build]
target = 'x86_64-unknown-linux-gnu'
```

Reply:
Ah right. Still I think it's better us managing it than users not knowing about this, or having to manage it anyway.

Reply:
but how? the only way is to write it or pass `--target` explicitly for host which both are not desirable IMO. I think this is a fair trade-off, like you only experience cache invalidation only when switching from desktop development to mobile development and vice versa which doesn't happen very often and won't affect CI anyways.

Reply:
oh god, please no

Reply:
`build.target` can also be a path to a "relative path to a .json target spec file". Maybe we could use that instead, and change the default template to gitignore that JSON file.

Reply:
IDK about that, seems like more workarounds and I'd rather keep the current one than go that way tbh.